appservice/src/SiteWrapper.ts

```diff
@@ -293,6 +298,43 @@ export class SiteWrapper {
         await requestP.get(`https://${this.defaultHostName}/admin/host/status`, requestOptions);
     }
 
+    public async validateDeployment(outputChannel: vscode.OutputChannel, pollingInterval: number = 5000, pollingAttempts: number = 12): Promise<responseStatus[]> {
```

Comment:
I don't think pollingInterval should be a parameter. It feels more like an internal implementation detail to me.

In any case, I think the pollingAttempts parameter is much more likely to be changed, so at least move that to be the first optional parameter